# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MacroAssembler.h
# Commit: 107ac71d3eb1
# Full Hash: 107ac71d3eb130663cd04b149c9d9d7143b7379e
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-30 16:09:44
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 7: Remove Frame::tls. r=lth
#   
#   
#   We remove Frame::tls and fix prologue/epilogue. Runtime, Ion and Baseline are ready for this move already.
#   
# ==============================================================================

diff -r 5737845e8af2 -r 107ac71d3eb1 js/src/jit/MacroAssembler.h
--- a/js/src/jit/MacroAssembler.h	Fri Oct 23 18:17:38 2020 +0000
+++ b/js/src/jit/MacroAssembler.h	Fri Oct 30 09:12:21 2020 +0000
@@ -3551,8 +3551,6 @@
   void guardStringToInt32(Register str, Register output, Register scratch,
                           LiveRegisterSet volatileRegs, Label* fail);
 
-  void loadWasmTlsRegFromFrame(Register dest = WasmTlsReg);
-
   template <typename T>
   void loadTypedOrValue(const T& src, TypedOrValueRegister dest) {
     if (dest.hasValue()) {